docs/sphinxext/github_link.py

```diff
@@ -0,0 +1,84 @@
+from operator import attrgetter
```

Comment:
This is totally my oversight, but would you mind adding something here to reference that this was taken from sklearn ? I believe their license is BSD, but I'm sure attribution is appreciated !

Reply:
Good catch! Should be good as of 6b1dfb7.